opm/parser/eclipse/EclipseState/SimulationConfig/tests/ThresholdPressureTest.cpp

```diff
@@ -181,13 +181,10 @@ static DeckPtr createDeck(const ParseContext& parseContext , const std::string&
     return parser.parseString(input , parseContext);
 }
 
-static Eclipse3DProperties getEclipseProperties(const Deck& deck)
-{
-    static const EclipseGrid eclipseGrid( 3, 3, 3 );
-    return Eclipse3DProperties( deck, TableManager( deck ), eclipseGrid );
+static Eclipse3DProperties getEclipseProperties(const Deck& deck) {
+    return Eclipse3DProperties(deck, *new TableManager(deck), *new EclipseGrid(3, 3, 3));
```

Comment:
It's in a test snippet, so the leak doesn't matter much. Anyway, the reason for the new is because the `static getEclipseProperties` is a shorter initialisation of a very common configuration used in the tests, but `3DProperties` relies on `TableManager` and `EclipseGrid` being alive. Instead of mandating 3 objects being constructed every time, just constructing-and-leaking here was deemed acceptable (by me).